src/corehost/common/pal.h

```diff
@@ -95,6 +95,9 @@ namespace pal
 
     inline int strcmp(const char_t* str1, const char_t* str2) { return ::wcscmp(str1, str2); }
     inline int strcasecmp(const char_t* str1, const char_t* str2) { return ::_wcsicmp(str1, str2); }
+    inline int strncmp(const char_t* str1, const char_t* str2, int len) { return ::wcsncmp(str1, str2, len); }
+    inline int strncasecmp(const char_t* str1, const char_t* str2, int len) { return ::_wcsnicmp(str1, str2, len); }
```

Comment:
`strcasecmp` is a funny name but used by unix, Ruby as `casecmp` and Python as `casefold`